WindowsServerDocs/storage/storage-spaces/Storage-Spaces-Direct-Hardware-Requirements.md

```diff
@@ -57,6 +57,7 @@ Note that RAM sizes use base-2 numbering (where 1 GB = 1,024 MB), while drives a
 For more help choosing drives, see the [Choosing drives](choosing-drives.md) topic.
 
 - Use local-attached SATA, SAS, or NVMe drives
+    - Note: fibre channel attached drives are not suported for 2012/2016 storage spaces.
```

Comment:
Thanks for the contribution! Could you move this to the end of the Not supported item so that it reads: 

**Not supported:** Multi-path IO (MPIO) or physically connecting drives via multiple paths. Fibre Channel drives also aren't supported in any topology.